stanford/bin/generate-certs-for-user

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+_required_variables='REMOTE_BOX COURSE_ID USER_NAME SYSTEM_USER'
```

Comment:
These should be set by jenkins, but you can set them manually when running directly via command line.